src/main/java/tr/com/srdc/cda2fhir/jolt/AdditionalModifier.java

```diff
@@ -205,6 +207,113 @@ protected Optional applyList(final List<Object> argList) {
 		}
 	}
 
+	private static String iterableJoin(Iterable<String> iterable, String delimiter) {
+		String joined = "";
+
+		Iterator<String> iterator = iterable.iterator();
+
+		while (iterator.hasNext()) {
+			joined += iterator.next();
+			if (iterator.hasNext())
+				joined += delimiter;
+		}
+
+		return joined;
+
+	}
+
```

Comment:
Clever but it already exists

List.stream().collect(Collectors.joining(delimiter))

Reply:
Nice, I was pretty surprised I couldn't find that but I guess I didn't look very hard.